server/routes/maintainingReports/admin.ts

```diff
@@ -0,0 +1,58 @@
+import type { RequestHandler } from 'express'
+import { paths } from '../../config/incident'
+import type { OffenderService, ReviewService } from '../../services'
+import type { SystemToken } from '../../types/uof'
+
+export default class AdminRoutes {
+  constructor(
+    private readonly reviewService: ReviewService,
+    private readonly offenderService: OffenderService,
+    private readonly systemToken: SystemToken
+  ) {}
+
+  viewEditReport: RequestHandler = async (req, res) =>
+    res.redirect(paths.editForm(Number(req.params.reportId), 'evidence'))
+
+  viewEditForm: RequestHandler = async (req, res) => {
+    const { reportId, formName } = req.params
+
+    const report = await this.reviewService.getReport(Number(reportId))
+    const { bookingId, reporterName, submittedDate, form } = report
+
+    const offenderDetail = await this.offenderService.getOffenderDetails(
+      await this.systemToken(res.locals.user.username),
+      bookingId
+    )
+
+    return res.render('pages/admin/edit-form.html', {
+      errors: req.flash('errors'),
+      data: {
+        incidentId: reportId,
+        reporterName,
+        submittedDate,
+        offenderDetail,
+        selectedFormName: formName,
+        formSections: Object.keys(form),
```

Comment:
should probably not use keys and instead order these appropriately and remove involved staff